src/components/ebay-calendar/component.js

```diff
@@ -106,7 +106,26 @@ export default class extends Marko.Component {
             input.disableList?.map(dateArgToISO) ?? []
         );
         if (this.isDisabled(this.state.tabindexISO)) {
-            this.state.tabindexISO = this.getFirstActiveISO(input);
+            const firstActive = this.getFirstActiveISO(input);
+            if (firstActive) {
+                this.state.tabindexISO = firstActive;
+            } else if (
+                this.state.disableBefore &&
+                this.state.tabindexISO < this.state.disableBefore
+            ) {
+                this.state.baseISO = this.state.disableBefore;
+                this.state.offset = 0;
+                this.state.tabindexISO =
+                    this.getFirstActiveISO(input) ?? this.state.disableBefore;
+            } else if (
```

Comment:
Is this `if/else if` exhaustive of all potential options? In other words, is there any other possibility outside of the conditions covered in the `if/else if`?

Reply:
There actually is an additional undocumented edge case that I left unaddressed here, which we may have to include as an anti-pattern. Suppose we have the following component:
```marko
<ebay-calendar disable-before="2023-06-30" disable-list=["2023-06-30"] />
```
In this case, there is still one day in June which is within the possible range, but that day has been _manually disabled_. I believe it is very unlikely that this problem will ever come up, but if it does the solution is to expand `disable-before`:
```marko
<ebay-calendar disable-before="2023-07-01" />
```

Reply:
If that is the only possible alternative, we should address it in an `else` block. and comment what condition that block is covering.

Reply:
I think this should be left as a fall-through case where we stay in the current month even though all days are disabled, because if we start searching for a day we may cause infinite loops unless we are very careful. I still added an `else` with documentation, but did not execute any code in it.

Reply:
Ok, that's a fair compromise. The comment in the `else` is helpful.